### DISCUSSION TITLE
Pro board updates

### DISCUSSION THREAD

**Discussion by frdfsnlght** [grblHAL/iMXRT1062#18] [General]
I believe Phil is supposed to be back from holiday but haven't heard anything in a while. I hope he's well and COVID free!

Any updates on the status of the new "Pro" board? I'm not seeing a Tindie store page yet. I think he said he had some boards back from the fab before he left on holiday but I expect those might be going to testers or some such? Any idea about general availability?

**Comment by terjeio** [comment]
Ping @phil-barrett - have you seen this?

**Comment by frdfsnlght** [comment]
Doh! I hadn't thought of doing that.
